keyboards/delikeeb/vanana/rev1/rules.mk

```diff
@@ -0,0 +1,23 @@
+# MCU name
```

Comment:
And same for `rules.mk`.

Reply:
Thank you. I will make changes accordingly.